qiskit_ibm_runtime/utils/utils.py

```diff
@@ -342,12 +342,10 @@ def default_runtime_url_resolver(
         elif channel == "ibm_quantum_platform":
             # ibm_quantum_platform url
             region = _location_from_crn(instance)
-            if region == "eu-de":
-                api_host = (
-                    f"{parsed_url.scheme}://{region}" f".quantum.{parsed_url.hostname}/api/v1"
-                )
-            else:
-                api_host = f"{parsed_url.scheme}://" f"quantum.{parsed_url.hostname}/api/v1"
+            region_prefix = "" if region == "us-east" else f"{region}."
+            api_host = (
+                f"{parsed_url.scheme}://{region_prefix}" f"quantum.{parsed_url.hostname}/api/v1"
+            )
```

Comment:
Special case handling should be done for `us-east` as opposed to for `eu-de` because there can be other non default urls that need to be appended.